AddShippingZone.tsx

```diff
@@ -1,18 +1,138 @@
-import React from "react";
-import { Button, SafeAreaView, Text, View } from "react-native";
+import React, { useState } from "react";
+import {
+  Linking,
+  Platform,
+  Pressable,
+  SafeAreaView,
+  ScrollView,
+  StyleSheet,
+  Text,
+  TextInput,
+  View,
+} from "react-native";
 import { useNavigation } from "@react-navigation/native";
+import CheckBox from "@react-native-community/checkbox";
+import { ToolbarActionButton } from "./ToolbarActionButton";
 
 const AddShippingZone = () => {
   const navigation = useNavigation();
 
+  const [isLimitEnabled, setLimitEnabled] = useState(false);
+
+  React.useEffect(() => {
```

Comment:
[Q] Do you need to use the `React` namespace in your setup? I'm curious because I don't need it in mine, should we keep adding it?

Reply:
I didn't import `useEffect` as it was like this in the navigation documentation 😅 didn't think about it twice. Let's keep the same style as you did: without `React.` prefix 👍